### PR TITLE
Feature routing

### PR DESCRIPTION
**Pull request by OmKakatkar**
#Features
* Adds basic routing
* Adds styling to header and sidebar
* Adds links to Sidebar
image

### CODE REVIEW

File: src/frontend/components/SingleVideo/SingleVideo.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { useParams } from 'react-router-dom';
+
+function SingleVideo() {
+	const { videoId } = useParams();
+	console.log(videoId);
```
Comment (rohit8686) [inline src/frontend/components/SingleVideo/SingleVideo.js]: better to remove console statement here

**Review by rohit8686 — APPROVED** [review]
Nice folder structure, looks good to merge

File: src/frontend/App/App.js [hunk 2]
```diff
@@ -3,20 +3,43 @@ import { ToastContainer } from 'react-toastify';
...
 		<div className="app">
 			<Header />
 			<SideBar />
 			<div className="main-container">
 				<Routes>
 					<Route path="/" element={<Home />} />
-					<Route path="/login" element={<Login />} />
-					<Route path="/signup" element={<SignUp />} />
 					<Route path="/mockman" element={<MockAPI />} />
+					{!user.token && (
```
Comment (VLeads) [inline src/frontend/App/App.js:29]: Appreciation: It's good that you have given condition to routing !!

**Review by VLeads — APPROVED** [review]
Looks good. You can merge

File: src/frontend/components/SingleVideo/SingleVideo.js (hunk 1 above)
Comment (OmKakatkar) [reply src/frontend/components/SingleVideo/SingleVideo.js]: done

File: src/frontend/App/App.js (hunk 2 above)
Comment (OmKakatkar) [reply src/frontend/App/App.js:29]: Thank You

**Pull request merged**